modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -2834,6 +2838,9 @@ void ONNXImporter::parseElementWise(LayerParams& layerParams, const opencv_onnx:
             if (layer_id.find(node_proto.input(i)) == layer_id.end())
             {
                 Mat inp = getBlob(node_proto, i);
+                if (i == 1) {
+                    inp.convertTo(inp, CV_32F);
+                }
```

Comment:
Same as above, can we check `if (inp.depth() != CV_32F)`;

Reply:
Done.